hugegraph-store/hg-store-core/pom.xml

```diff
@@ -121,9 +121,48 @@
             <version>${revision}</version>
             <scope>system</scope>
             <systemPath>
-                ${project.basedir}/../hg-store-dist/src/assembly/ext-lib/hugegraph-core-1.5.0.jar
+                ${top.level.dir}/hg-store-dist/src/assembly/ext-lib/hugegraph-core-1.5.0.jar
             </systemPath>
         </dependency>
+        <!-- tinkerpop -->
+        <dependency>
```

Comment:
why we need add tinkerpop dependency here?

@VGalaxies any context?

Reply:
As mentioned in the last item of "Main change", tinkerpop is depended on by `hg-store-core`, but in internal version the dependency is imported through `hugegraph-core`. The current temporary solution is that importing `hugegraph-core` jar package through system dependency, whose dependencies cannot be passed on, so the dependencies need to be added again in `hg-store-core`

Reply:
> why we need add tinkerpop dependency here?
> 
> @VGalaxies any context?

As stated by @heiyan-2020, assuming there are projects A (`hg-store-core`) and B (`hugegraph-core`), where project A depends on project B, and project B is a system dependency, the dependencies within project B will not be automatically inherited by project A.

Reply:
> As mentioned in the last item of "Main change", tinkerpop is depended on by `hg-store-core`, and in internal version the dependency is passed on through `hugegraph-core` which imported as usal maven dependency. But the current temporary solution is that importing `hugegraph-core` jar package through system dependency. In this way, the dependencies in `hugegraph-core` cannot be passed on, so the dependencies need to be added again in `hg-store-core`

@heiyan-2020 get it, but we need add `comment/TODO` mark here? (maybe in next PR)